scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -63,7 +63,7 @@ if [[ "${DOGRI_WAV}" = "YES" ]]; then
 fi
 
 # Check for input templates for grib2 products (copying will be done in the grib2 script)
-if [[ "${DOGRB_WAV}" = "YES" ]]; then
+if [[ "${DOGRB_WAV}" == "YES" ]]; then
   for grbGRD in ${waveinterpGRD} ${wavepostGRD}; do
     if [[ ! -f "${PARMgfs}/wave/ww3_grib2.${grbGRD}.inp.tmpl" ]]; then
       echo "FATAL ERROR: No template for grib generation"
```

Comment:
Should this FATAL ERROR have an exit/err_chk?

Reply:
Yes it should.  Setting `DOGRB_WAV=NO` raises a `FATAL ERROR` later anyway.  It would be better to fail out now.

Reply:
Fixed.